### ISSUE TITLE
unreserve outputs before sending sweep transaction

### ISSUE THREAD

**Issue by ubbabeck** [breez/breez-sdk#531]
In order to replace a transaction if eg. fee is too low when transferring from the internal wallet.  So it the output is reserved it would be nice to unreserve so the user can broadcast the new transaction. 

Ref c-breez issue https://github.com/breez/c-breez/issues/676.

**Comment by dangeross** [comment]
Seems the only way to unreserve inputs is using the psbt result of the withdraw. So we need a mechanism to find or cache this tx/psbt

**Comment by roeierez** [comment]
@dangeross  I see. The problem is that there could be multiple psbt created in the past each for a set of utxos...
Perhaps we should use a different way to withdraw? create a new psbt (https://docs.corelightning.org/reference/lightning-utxopsbt) and then send it using https://docs.corelightning.org/reference/lightning-signpsbt and https://docs.corelightning.org/reference/lightning-sendpsbt ?
I think this process allows using reserved inputs.

**Comment by dangeross** [comment]
Yes, it's possible to use lightning-utxopsdt with already reserved inputs. There is also the option to not reserve the input utxos used to create the psbt. So we could in theory only use not reserved inputs and not reserve them. Something to think about.

**Comment by dangeross** [comment]
Currently blocked by [lightning-addpsbtoutput](https://docs.corelightning.org/reference/lightning-addpsbtoutput) not being available over cln-grpc

**Comment by erdemyerebasmaz** [comment]
@dangeross It looks like `AddPsbtOutput` is available over [cln-grpc](https://github.com/ElementsProject/lightning/tree/master/cln-grpc) now.

CC: @roeierez
